src/client/theme-default/components/VPNavBar.vue

```diff
@@ -52,8 +52,8 @@ const { hasSidebar } = useSidebar()
   position: relative;
   border-bottom: 1px solid var(--vp-c-divider-light);
   padding: 0 8px 0 24px;
-  height: var(--vp-nav-height-mobile);
-  transition: border-color 0.5s, background-color 0.5s;
+  min-height: calc(var(--vp-nav-height-mobile) + 1px);
+  transition: background-color 0.5s;
```

Comment:
Transitioning the border-color made the border flash from white to its final dark color when using the dark theme and resizing the window (between large and medium layouts). Seen in Chrome and Firefox.

I think the goal was to have a smoother transition when changing the theme (in the small and medium layouts). The result for that without the border-color transition was not shocking to my eyes, unlike the white-to-dark flash on resize. But I can restore this transition setting if need be.